packages/docusaurus-theme-classic/src/theme/StructuredData/index.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import type {Props} from '@theme/StructuredData';
+
+export default function StructuredData({structuredData}: Props): JSX.Element {
```

Comment:
that would be better in theme-common, there's no good reason to swizzle that comp IMHO and it's not related to the theme/styling but is a reusable util

Reply:
I tried to move this across, but got stuck when it came to migrating the types.  Whilst there's a `theme-classic.d.ts`, there doesn't appear to be a `theme-common.d.ts` - couldn't work out what I should do.  Happy for this to move, but could you move this please?

Reply:
Never mind - think I've sorted it